src/FreeDSx/Snmp/Module/SecurityModel/UserSecurityModelModule.php

```diff
@@ -33,6 +34,7 @@
 use FreeDSx\Snmp\Request\GetRequest;
 use FreeDSx\Snmp\Request\TrapV2Request;
 use FreeDSx\Snmp\Response\ReportResponse;
+use function assert;
```

Comment:
The assert imports can be removed now I guess. Wasn't sure if you planned to do that during the rebase. Figured I'd mention it.

Reply:
sure, forgot it